src/modules/collection/types.ts

```diff
@@ -41,3 +41,4 @@ export type Access = {
 }
 
 export const COLLECTION_NAME_MAX_LENGTH = 32
+export const TP_COLLECTION_NAME_MAX_LENGTH = 64
```

Comment:
Should we be using this length on the EditCollectionNameModal? it was generic up until this point but it uses `COLLECTION_NAME_MAX_LENGTH`

Reply:
The UI scares me a bit, things are a bit cramped up:

Current:
<img width="1055" alt="Screen Shot 2022-03-14 at 12 24 53" src="https://user-images.githubusercontent.com/692077/158204867-a3b38fc5-96ec-4667-9b2c-02adb23ad33a.png">


Longer:

<img width="1217" alt="image" src="https://user-images.githubusercontent.com/692077/158204836-37b43fee-7122-426f-b937-43cfbcfcadf5.png">

Reply:
Good call! I updated the modal name to use `TP_COLLECTION_NAME_MAX_LENGTH` as the max length for TPs. 
Also, I updated the styles so the name container doesn't overflow when it's too long:

<img width="1176" alt="image" src="https://user-images.githubusercontent.com/8763687/158348651-0b93b779-5384-43aa-80a7-cc9206ac4c37.png">

Reply:
![telegram-cloud-photo-size-1-5010286617772009953-y](https://user-images.githubusercontent.com/692077/158467323-5687282b-391d-4769-a88b-cd02a61c325c.jpg)